auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+            
+            TokenBlacklist.objects.create(
+                token=token,
+                token_type=token_type,
+                user=user,
+                reason=reason,
+                expires_at=expires_at
+            )
+            
+            logger.info(
+                f"Token blacklisted: type={token_type}, "
+                f"user={user.username}, reason={reason}"
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)